labonneboite/importer/scripts/create_fake_alternance_hirings.sql

```diff
@@ -0,0 +1,12 @@
+# this script should only be used in development environnement, not in lbbdev nor in test
+
+# FIXME later when we regularly receive new alternance data,
+# implement proper "check_alternance" and "extract_alternance" jenkins jobs
+
+# duplicate CDD hirings as APR hirings
+insert into hirings(siret, hiring_date, departement, contract_type)
+select siret, hiring_date, departement, 11 from hirings where contract_type=1;
+
+# duplicate CDI hirings as CP hirings
+insert into hirings(siret, hiring_date, departement, contract_type)
+select siret, hiring_date, departement, 12 from hirings where contract_type=2;
```

Comment:
pin.: mettre les mots clé SQL en lettres capitales pour rendre le script plus lisible ?

Reply:
en fait j'écris en minuscules partout ailleurs (code, tickets, études etc) préférence personnelle je reconnais.

Franchement en python si on exigeait que tu capitalises les mots clés FOR, IF, DEF, tu trouves vraiment que ça ajouterait qqch?

Reply:
En python, les chaînes de caractères sont bien différenciées des variables par des guillemets, ce qui n'est pas le cas en SQL.

Reply:
euh en SQL aussi les chaines de caractères sont entourées de guillemets, ce qui permet de les différencier des noms de variables

```
select 18, 19, 'abc';

select abc;  // FAIL sauf s'il existe une colonne abc
```

Reply:
parfois les noms de colonnes sont mis entre backquotes (en cas de caractères spéciaux) mais c'est différent des guillemets